bindata/network/ovn-kubernetes/common/ipsec.yaml

```diff
@@ -105,7 +105,7 @@ spec:
             counter=0
             until [ ! -z $(kubectl get csr/$(hostname) -o jsonpath='{.status.certificate}' 2>/dev/null) ]
             do
-              ((counter++))
+              counter=$[counter+1]
```

Comment:
can you use `$((counter + 1))` instead? `$[]` is apparently some archaic bash syntax that I've never seen before. `$(())` is standard